x-pack/plugins/apm/public/components/alerting/ui_components/chart_preview/index.tsx

```diff
@@ -64,36 +67,88 @@ export function ChartPreview({
     opacity: thresholdOpacity,
   };
 
+  const DEFAULT_DATE_FORMAT = 'Y-MM-DD HH:mm:ss';
+
+  const tooltipProps: TooltipProps = {
+    headerFormatter: ({ value }) => {
+      const dateFormat =
+        (uiSettings && uiSettings.get(UI_SETTINGS.DATE_FORMAT)) ||
+        DEFAULT_DATE_FORMAT;
+      return moment(value).format(dateFormat);
+    },
+  };
+
+  const filteredSeries = useMemo(() => {
+    const sortedSeries = series.sort((a, b) => {
+      const aMax = Math.max(...a.data.map((point) => point.y as number));
+      const bMax = Math.max(...b.data.map((point) => point.y as number));
+      return bMax - aMax;
+    });
+    return sortedSeries.slice(0, NUM_BUCKETS_FOR_PREVIEW_CHART);
```

Comment:
Can you use lodash `sortBy` for this?

Reply:
We need to sort the series based on the Y coordinate in `series.data` in descending order. We are looking at top 5 groups with max Y values. I need to check how to apply lodash `sortBy` for this use case.

Do you have a suggestion on how to apply it? Otherwise I would like to refactor this in a separate PR.